### ISSUE TITLE
发布一个issue

### ISSUE THREAD

**Issue by qianphong** [qianphong/vuetify_style#1]
> loren

```vue
<template>
  <div>{{ name }}</div>
</template>
<script lang="ts" setup>
import { ref } from 'vue'
const name = ref('rick')
</script>
```

表头|表头|表头
--- | :--: | ---:
内容|内容|内容
内容|内容|内容

![name](https://assets.gitee.com/assets/issues/pic_issue_default-f9c4dd2dbbe1797c75f1eae0bf54d0d7.png)

```diff
+ 123123
console.log(1)
- 1231231
```
![profile-pic (4)](https://user-images.githubusercontent.com/44841842/133587040-bab90050-f644-48b5-84c0-8739a6bbcdc8.png)

https://user-images.githubusercontent.com/44841842/133587233-23c2b99a-0ca0-42fa-941a-92670a901374.mp4

**Comment by qianphong** [comment]
> > loren
> 
> ```
> <template>
>   <div>{{ name }}</div>
> </template>
> <script lang="ts" setup>
> import { ref } from 'vue'
> const name = ref('rick')
> </script>
> ```
> 
> 表头	表头	表头
> 内容	内容	内容
> 内容	内容	内容
> ![name](https://camo.githubusercontent.com/a0d656bdfcb893d9b8131868199b16ccc24a84362f18dd602971eefb1c58f4c2/68747470733a2f2f6173736574732e67697465652e636f6d2f6173736574732f6973737565732f7069635f69737375655f64656661756c742d66396334646432646262653137393763373566316561653062663534643064372e706e67)
> 
> ```diff
> + 123123
> console.log(1)
> - 1231231
> ```
> 
> ![profile-pic (4)](https://user-images.githubusercontent.com/44841842/133587040-bab90050-f644-48b5-84c0-8739a6bbcdc8.png)
> 
>  default.mp4

yes

**Comment by qianphong** [comment]
do you know [#1](https://github.com/qianphong/vuetify_style/issues/1#issue-997990343)

**Comment by qianphong** [comment]
#1

**Comment by qianphong** [comment]
https://github.com/qianphong/vuetify_style/commit/11c85d3f75d3d431524daae831510b5d49a46d2f

**Comment by qianphong** [comment]
#2

**Comment by qianphong** [comment]
closed #2
